### DISCUSSION TITLE
"Guides Index" Subnav - Release Notes spillover

### DISCUSSION THREAD

**Discussion by claudioduarte** [rails/rails#51530] [Docs]
Found this issue today on the homepage when I click on **Guides Index**. The `Release Notes` spillover the subnav/dropdown menu

![Screenshot 2024-04-09 at 16 28 12](https://github.com/rails/rails/assets/4729521/966ac603-8ff6-4aaf-9ac0-26dd09b4c641)

**Comment by claudioduarte** [comment]
Looks like this is happening on Brave and Chrome browsers. I just checked Safari and everything is in order.

**Comment by AmandaPerino** [comment]
Thanks @claudioduarte! We will take a look. cc: @jathayde

**Comment by carlosantoniodasilva** [comment]
This was just fixed on main. https://github.com/rails/rails/pull/51532
